docs/merkletree/treetex/main.go

```diff
@@ -186,9 +186,16 @@ func drawLeaf(prefix string, index uint64, leafText, dataText nodeTextFunc) {
 	a := nInfo[id]
 
 	// First render the leaf node of the Merkle tree
+	rangeIndices := a.rangeIndices
```

Comment:
Could copy the whole `a` here, and use it for the second `Printf`.

Reply:
good call, done.